packages/server/src/asyncQuestion/asyncQuestion.controller.ts

```diff
@@ -269,12 +269,6 @@ export class asyncQuestionController {
 
     const updatedQuestion = await question.save();
 
-    // Update Redis queue
-    await this.redisQueueService.updateAsyncQuestion(
-      `c:${question.course.id}:aq`,
-      updatedQuestion,
-    );
-
```

Comment:
erm, why are we deleting this? Won't this mean that when you update an async question, it won't display the update since redis won't know of the new question?

Reply:
okay turns out this is duplicate logic below